docs/se/config/advanced-configuration.adoc

```diff
@@ -460,37 +468,37 @@ Config config = Config.create(
 
 <1> Uses the `parserMapping` method to map keys to parser instances.
 <2> Tells the config system to use the HOCON parser for translating the `String`
-value of the `app` key. (HCON is a superset of JSON.)
+value of the `app` key. (HOCON is a superset of JSON.)
 
 As before, the config system replaces the value node in the
 containing config tree with the config tree resulting from the additional parse.
 
-=== Loading Config using Meta-configuration [[Config-Advanced-metaconfig]]
-
-This section is now available in xref:config-profiles.adoc[config profiles],
- which make meta-configuration obsolete - you can achieve the same with less configuration.
-A profile file is a meta-configuration file selected by the rules defined in the link above.
-
+== Loading Config using Meta-configuration [[Config-Advanced-metaconfig]]
+[IMPORTANT]
+=========
+You can achieve this with less configuration using xref:config-profiles.adoc[config profiles],
```

Comment:
Config profiles and meta configuration are the same thing in Helidon SE

Reply:
removed reference from table as well as section from advanced doc.